lightning/src/ln/features.rs

```diff
@@ -427,7 +493,6 @@ impl<T: sealed::Context> Features<T> {
 		Features::<C> { flags, mark: PhantomData, }
 	}
 
-	#[cfg(test)]
```

Comment:
Can we expand the documentation to say "little endian" and note that "this is byte-swapped from most on-the-wire encodings"?